sdks/python/apache_beam/runners/interactive/interactive_runner.py

```diff
@@ -140,6 +151,10 @@ def run_pipeline(self, pipeline, options):
     result = pipeline_to_execute.run()
     result.wait_until_finish()
 
+    if result.state is beam.runners.runner.PipelineState.DONE:
```

Comment:
Can you test this please?

Reply:
Added a test that runs a pipeline, verifies the computed PCollections and checks the result of computed PCollections.